pkg/standalone/standalone.go

```diff
@@ -1061,18 +1061,27 @@ func downloadFile(dir string, url string) (string, error) {
 	fileName := tokens[len(tokens)-1]
 
 	filepath := path.Join(dir, fileName)
-	_, err := os.Stat(filepath)
+
+	print.InfoStatusEvent(os.Stdout, "  => download to: %s", filepath)
+
+	fileStat, err := os.Stat(filepath)
```

Comment:
This should not work because if Dapr already exists in `filepath`, `init` should fail before https://github.com/dapr/cli/blob/8b68de52da469894dff77978e939b634677eb4bf/pkg/standalone/standalone.go#L139